vue/mixins/logic.js

```diff
@@ -26,8 +30,6 @@ export const logicMixin = {
          * model, version and parts defined in instance.
          */
         async configRipe() {
-            if (!this.config) return;
-

```

Comment:
@gcandal @joamag I don't think this removel is correct, because `configRipe` is used directly by the components and not just in `setupRipe`
Example:
[here](https://github.com/ripe-tech/ripe-sdk-components-vue/blob/daa5deb942acf424620e13577a01e11c01a68a62/vue/components/organisms/ripe-image/ripe-image.vue#L214), [here](https://github.com/ripe-tech/ripe-sdk-components-vue/blob/daa5deb942acf424620e13577a01e11c01a68a62/vue/components/organisms/ripe-configurator/ripe-configurator.vue#L324), [here](https://github.com/ripe-tech/ripe-sdk-components-vue/blob/daa5deb942acf424620e13577a01e11c01a68a62/vue/components/organisms/ripe-price/ripe-price.vue#L118) and [here](https://github.com/ripe-tech/ripe-sdk-components-vue/blob/daa5deb942acf424620e13577a01e11c01a68a62/vue/components/organisms/ripe-price/ripe-price.vue#L135)

Reply:
@NFSS10 add the `if (!this.config) return;` then. It might seem code repetition, but semantically it sounds better than calling `doA` and then it doesn't do `A`.